examples/translation-job.json

```diff
@@ -0,0 +1,17 @@
+{
+    "id": "cf5e1c7c-86ae-4f03-aa9e-513b46cd3b7e",
+    "created_on": "2016-10-11T12:11:32+02:00",
+    "created_by": "Ivan Wolf",
+    "item": {
+        "type": "article",
+        "id": "051273",
+        "field": "character-fr",
+        "name": "Mandorla Primitivo IGT Puglia"
+    },
+    "source_language": "de",
+    "source_text": "Dunkles Rubinrot. In der Nase Aromen von roten Beeren, reifen Kirschen und Brombeeren. Geschmeidig im Gaumen.",
+    "translated_language": "fr",
+    "translated_on": "2016-10-13T06:12:37+02:00",
+    "translated_by": "Mirjam Dupond",
+    "translated_text": "Robe rubis foncé. Nez de baies rouges, de cerises matures et de mûres. Bouche moelleuse."
```

Comment:
Should we add a status? Easier filtering, extenable in the future when there are more than the two (implicit) states (not translated, translated).